packages/prisma/src/prisma/migrations/20230827124621_media_files/migration.sql

```diff
@@ -0,0 +1,45 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Attachment` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `AttachmentInstance` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Attachment" DROP CONSTRAINT "Attachment_ownerId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Attachment" DROP CONSTRAINT "Attachment_questionId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "AttachmentInstance" DROP CONSTRAINT "AttachmentInstance_questionInstanceId_fkey";
```

Comment:
These modifications do not really make sense, if we drop the `Attachment` and `Attachment tables afterwards anyways, right?`

Reply:
these files are generated, I think you need to drop constraints before you can delete a postgres table, but not sure :) just what prisma does.